src/content/english.js

```diff
@@ -218,44 +218,50 @@ const TESTIMONIALS = [
         date: 'President & CTO',
-        portrait: 'https://user-images.githubusercontent.com/4914611/63651576-6e1cb200-c756-11e9-88f2-87cb80faa5a1.png'
+        portrait: 'https://user-images.githubusercontent.com/49292858/68433886-0808ce00-018e-11ea-9818-64cd4372bdc6.png'
+    },
+    {
+        name: 'Dave Corkle',
+        quote: '',
+        date: 'Director',
+        portrait: ''
     },
     {
         name: 'Viki Lafarga',
         quote: 'The SetLife Network’s project-based learning trial was a unique opportunity for me in that I was both learning about and gaining work experience in several areas at once: corporate governance, programming and design, productivity, team management, and more. I am particularly interested in how the different areas that make up an organization interact together and in working at those intersections, so I was grateful for a work environment where I had to sync up with my team every week and learn about what else was going on . And not just what my coworkers were doing but also what obstacles they were coming up against, which I think makes for more solid teamwork. That’s really not something most people get to experience in a society where work is mostly specialized into separate departments.',
         date: 'Dir. of International Operations',
-        portrait: 'https://media.licdn.com/dms/image/C4D03AQHFpkfc8bQt5g/profile-displayphoto-shrink_800_800/0?e=1572480000&v=beta&t=-J7whAKcGixu0JyYzmVFXJgxPyD667cw5qNq580GmMA'
+        portrait: 'https://user-images.githubusercontent.com/49292858/68434040-5fa73980-018e-11ea-9ab5-079506b65ad2.jpg'
     },
     {
         name: 'Quinn Pruitt',
         quote: 'Project based learning is by far the most effective learning environment for software development. In seven weeks, I learned more about backend development and product management than I would have learned in an entire semester in college. The projects were quite challenging, yet incredibly rewarding. The high level of autonomy and personal responsibility provided a strong sense of ownership for my work, which I found crucial to my growth and success. I would recommend this program to anyone looking to level up their professional skills in a fun an engaging learning environment.',
         date: 'Technical Advisor',
-        portrait: 'https://media.licdn.com/dms/image/C5603AQEtPbVdzw90IQ/profile-displayphoto-shrink_800_800/0?e=1572480000&v=beta&t=FQ4lOzdfOXfou0PvkotYPpP59hdqDXFEJsntss3b7q4'
+        portrait: 'https://user-images.githubusercontent.com/49292858/68435657-39839880-0192-11ea-95ab-43c9f1bd8c91.jpg'
```

Comment:
Use this image instead: https://user-images.githubusercontent.com/4914611/68456387-e7785c80-01fd-11ea-91ee-14dec845c871.jpeg